Source/non_ip/ai.conf

```
@@ -17,6 +17,14 @@ DOMAIN-KEYWORD,openai
 # DOMAIN,openaicomproductionae4b.blob.core.windows.net
 # DOMAIN,production-openaicom-storage.azureedge.net
 
+# >> Apple Intelligence
+DOMAIN-SUFFIX,smoot.apple.com
+DOMAIN,apple-relay.apple.com
+DOMAIN,apple-relay.cloudflare.com
+DOMAIN,apple-relay.fastly-edge.com
```

```suggestion
DOMAIN,apple-relay.apple.com
DOMAIN,apple-relay.cloudflare.com
DOMAIN,apple-relay.fastly-edge.com
```

`apple-relay` domains use the OHTTP Protocol. This protocol is designed to prevent the exposure of client IPs, so it is impossible to perform Geoblocking through OHTTP Relay.

https://docs.fastly.com/products/oblivious-http-relay
https://blog.cloudflare.com/stronger-than-a-promise-proving-oblivious-http-privacy-properties/

---

These lines are also existed in [domainset/cdn.conf](https://github.com/SukkaW/Surge/blob/master/Source/domainset/cdn.conf). I think we could move these lines to  [apple_cdn.conf](https://github.com/SukkaW/Surge/blob/master/Source/non_ip/apple_cdn.conf) instead.